HighscoreWindow.cpp

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2022, Harshit Sharma <harshits908@gmail.com>
+ * This file is distributed under the terms of the MIT license
+ */
+
+#include "HighscoreWindow.h"
+
+#include "WindowBoard.h"
+
+#include <Catalog.h>
+#include <string>
+#include <LayoutBuilder.h>
+#include <Message.h>
+#include <String.h>
+#include <StringView.h>
+
+#undef B_TRANSLATION_CONTEXT
+#define B_TRANSLATION_CONTEXT "HighscoreWindow"
+
+HighscoreWindow::HighscoreWindow(const char* oldHighscorer, const int32 oldHighscore, const int32 newHighscore)
+	:
+	BWindow(BRect(200, 200, 650, 600), B_TRANSLATE_SYSTEM_NAME("Enter you High Score"), B_TITLED_WINDOW, 0)
+{
+	BStringView* congratulations = new BStringView("congratulations", B_TRANSLATE(" "));
+	BString congratsText = "Congratulations!";
+	congratulations->SetText(congratsText.String());
```

Comment:
I don't understand why you are writing so complicated code here?

- B_TRANSLATE(" ") is asking translators to translate a string with nothing in it
- On the other hand, the "Congratulations!" text cannot be translated
- Why create a BString if in the next line all you do with it is convert it back to a normal C string? why not pass the string directly to SetText?

Reply:
I see, that got really complicated without any utility. Changing it to meaningful code.